src/browser/ui/dom/HTMLDOMPropertyConfig.js

```diff
@@ -104,6 +104,7 @@ var HTMLDOMPropertyConfig = {
     httpEquiv: null,
     icon: null,
     id: MUST_USE_PROPERTY,
+    inputMode: null,
```

Comment:
I was just testing this and it appears that we must use get/setAttribute. With `null` here we'll use the property which doesn't seem to work.

Did you test your local build?